src/styles/styles.js

```diff
@@ -760,26 +764,35 @@ const styles = {
         minHeight: variables.componentSizeNormal,
     },
 
-    textInputCompose: addOutlineWidth({
-        borderWidth: 0,
-        borderRadius: 0,
-        height: 'auto',
-        minHeight: 38,
-        paddingTop: 10,
-        paddingRight: 8,
-        paddingBottom: 10,
-        paddingLeft: 8,
-    }, 0),
+    textInputCompose: addOutlineWidth(
+        {
+            borderWidth: 0,
+            borderRadius: 0,
+            height: 'auto',
+            minHeight: 38,
+            paddingTop: 10,
+            paddingRight: 8,
+            paddingBottom: 10,
+            paddingLeft: 8,
+        },
+        0,
+    ),
 
     chatItemSubmitButton: {
         alignSelf: 'flex-end',
         borderRadius: 6,
         height: 32,
         paddingTop: 8,
         paddingRight: 6,
-        paddingBottom: 8,
+
+        // paddingBot   tom: 8,
+        
         paddingLeft: 6,
         margin: 3,
+
+        //condition can be removed if this padding issue affect both platforms (Android & IOS)ds
+       
+        marginBottom: Platform.OS === 'android' ? 8 : 0, 
```

Comment:
We prefer to use platform-specific extensions to avoid many checks to `Platform.OS`. Therefore we do not have checks like this in the app. Please refer to this documentation thanks.

https://github.com/Expensify/Expensify.cash#platform-specific-file-extensions